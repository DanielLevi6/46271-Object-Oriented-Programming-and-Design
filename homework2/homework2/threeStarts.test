# threeStarts test file

# Create a graph
CreateGraph myGraph

# Add nodes
CreateNode node1 4
CreateNode node2 5
CreateNode node3 2
CreateNode node4 1
CreateNode node5 3
CreateNode node6 4

AddNode myGraph node1
AddNode myGraph node2
AddNode myGraph node3
AddNode myGraph node4
AddNode myGraph node5
AddNode myGraph node6

# Add an edge
AddEdge myGraph node1 node2
AddEdge myGraph node2 node3
AddEdge myGraph node2 node4
AddEdge myGraph node3 node4
AddEdge myGraph node4 node5
AddEdge myGraph node4 node6

# Find a path
FindPath myGraph node1 node2 node3 -> node6 
