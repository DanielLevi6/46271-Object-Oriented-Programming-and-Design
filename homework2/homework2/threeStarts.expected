# threeStarts test file

# Create a graph
created graph myGraph

# Add nodes
created node node1 with cost 4
created node node2 with cost 5
created node node3 with cost 2
created node node4 with cost 1
created node node5 with cost 3
created node node6 with cost 4

added node node1 to myGraph
added node node2 to myGraph
added node node3 to myGraph
added node node4 to myGraph
added node node5 to myGraph
added node node6 to myGraph

# Add an edge
added edge from node1 to node2 in myGraph
added edge from node2 to node3 in myGraph
added edge from node2 to node4 in myGraph
added edge from node3 to node4 in myGraph
added edge from node4 to node5 in myGraph
added edge from node4 to node6 in myGraph

# Find a path
shortest path in myGraph: node3 node4 node6
